client/src/components/interests.js

```diff
@@ -1,7 +1,7 @@
 import { Box, Flex, Heading, extendTheme } from '@chakra-ui/react';
 import React from 'react';
 import { Header } from './header';
-
+import Preferences from './preferences/Preferences.jsx';
```

Comment:
Is the preferences component being rendered anywhere? Or could `interests.js` be replaced with `Preferences.jsx`?

Reply:
Preferences is a component which is rendered within the route interests.

Reply:
Gotcha. I assume a future PR will render it inside the interests page since you're importing it here?

Reply:
It's there. I've already sent and update. Somehow I've deleted it before. Thanks for catching it.

Reply:
No problem, looks good now :)